src/deploymentstatus.h

```diff
@@ -52,4 +52,10 @@ inline bool DeploymentEnabled(const Consensus::Params& params, Consensus::Deploy
     return params.vDeployments[dep].nTimeout != 0;
 }
 
+/** this function is convenient helper for DIP0003 because 'active' and 'enforced' are different statuses for DIP0003 */
+constexpr inline bool DeploymentDIP0003Enforced(const int nHeight, const Consensus::Params& params)
```

Comment:
nit: constexpr is implicitly inline https://stackoverflow.com/questions/14391272/does-constexpr-imply-inline
```suggestion
constexpr bool DeploymentDIP0003Enforced(const int nHeight, const Consensus::Params& params)
```